src/extension-test/unit/cursor-doc/paredit-test.ts

```diff
@@ -1662,9 +1662,7 @@ describe('paredit', () => {
         expect(textAndSelection(a)).toEqual(textAndSelection(b));
       });
 
-      // NB: enabling this breaks bunch of other tests.
-      //     Not sure why, but it can be run successfully by itself.
-      xit('splice string', async () => {
+      it('splice string', async () => {
```

Comment:
This is plain wonderful to be able to enable again. It has bugged me to no end why one test would mess up for other tests, but your fix makes it obvious why that could happen.  Thanks, @bpringe! ❤️ 🎉

Reply:
Awesome! Glad to help.